src/svg2pdf.js

```diff
@@ -1885,11 +1885,12 @@ SOFTWARE.
       attributeState.fontSize = parseFloat(fontSize);
     }
 
-    var verticalAlign = getAttribute(node, "vertical-align") || getAttribute(node, "alignment-baseline");
-    if (verticalAlign) {
-      attributeState.verticalAlign = verticalAlign.split(" ").find(function (keyWord) {
-        return ["baseline", "text-bottom", "alphabetic", "ideographic", "middle", "central", "mathematical", "text-top", "bottom", "center", "top", "hanging"].includes(keyWord);
-      });
+    var alignmentBaseline = getAttribute(node, "vertical-align") || getAttribute(node, "alignment-baseline");
+    if (alignmentBaseline) {
+      var matchArr = alignmentBaseline.match(/(baseline|text-bottom|alphabetic|ideographic|middle|central|mathematical|text-top|bottom|center|top|hanging)/);
+      if (isArray(matchArr)) {
```

Comment:
or simply `if (match)`. Where is `isArray` defined? Usually, it's `Array.isArray`.